docs/src/getting-started.md

```diff
@@ -1,104 +1,174 @@
 # **2.** Getting Started
 
-Pkg comes with its own REPL mode, which can be entered from the Julia REPL
-by pressing `]`. To get back to the Julia REPL press backspace or ^C.
+What follows is a quick overview of Pkg, Julia's package manager.
+It should help new users become familiar with basic Pkg features.
 
-```
-(v1.1) pkg>
-```
+## Basic Usage
 
-The prompt displays the current active environment, which is the environment
-that Pkg commands will modify. The default environment is based on the current
-Julia version, `(v1.1)` in the example above, and is located in
-`~/.julia/environments/`.
+Pkg comes with a REPL.
+Enter the Pkg REPL by pressing `]` from the Julia REPL.
+To get back to the Julia REPL, press backspace or ^C.
 
 !!! note
-    There are other ways of interacting with Pkg:
-    1. The `pkg` string macro, which is available after `using Pkg`. The command
-       `pkg"cmd"` is equivalent to executing `cmd` in the Pkg REPL.
-    2. The API-mode, which is recommended for non-interactive
-       environments. The API mode is documented in full in the API Reference section
-       of the Pkg documentation.
+    This guide relies on the Pkg REPL to execute Pkg commands.
+    For non-interactive use, we recommend the Pkg API.
+    The Pkg API is fully documented in the API Reference section of the Pkg documentation.
 
+Upon entering the Pkg REPL, you should see a similar prompt:
 
-To add a package, use the `add` command
+    (v1.1) pkg>
```

Comment:
For consistency with the rest of the manual I would prefer using fenced blocks like before.

Reply:
Fair point. I'll change it back :+1: